bin/add_frontend_server.rb

```diff
@@ -0,0 +1,120 @@
+require 'aws-sdk'
+require_relative '../deployment'
+
+MAX_WAIT_TIME = 600 #Wait no longer than 10 minutes for instance creation. Typically this takes a couple minutes
+
+Aws.config.update({
+                      region: CDO.aws_region,
+                      credentials: Aws::Credentials.new(CDO.aws_access_key, CDO.aws_secret_key),
+                  })
```

Comment:
Since #3995 has been merged, you can probably delete these four lines and the behavior should stay the same.